salt/iiif/config/etc-loris2-loris2.conf

```diff
@@ -16,109 +16,92 @@
 #
 # <http://www.voidspace.org.uk/python/configobj.html#unrepr-mode>
 #
-# String interpolation is enabled using the "template" style. OS environment
-# variables are available for interpolation, e.g., run_as_user='$USER'
-#
-# <http://www.voidspace.org.uk/python/configobj.html#string-interpolation>
+# String interpolation is disabled.
 #
 
 [loris.Loris]
-    tmp_dp = '{{ pillar.iiif.loris.storage }}/tmp'
-    www_dp = '/var/www/loris2'
-    run_as_user = 'loris'
-    run_as_group = 'loris'
-    enable_caching = True
-    redirect_canonical_image_request = False
-    redirect_id_slash_to_info = True
-    
-    # max_size_above_full restricts interpolation of images on the server.
-    # Default value 200 means that a user cannot request image sizes greater than
-    # 200% of original image size (width or height).
-    # Set this value to 100 to disallow interpolation. Set to 0 to remove
-    # size restriction.
-    max_size_above_full = 200
-
-#proxy_path=''
-# cors_regex = ''
-# NOTE: If supplied, cors_regex is passed to re.search():
-#    https://docs.python.org/2/library/re.html#re.search
-# Any url_root:
-#    http://werkzeug.pocoo.org/docs/latest/wrappers/#werkzeug.wrappers.BaseRequest.url_root
-# (i.e., https?://domain.edu(:port)?/) that matches will be
-# set to the value of Access-Control-Allow-Origin.
+tmp_dp = '{{ pillar.iiif.loris.storage }}/tmp' # r--
```

Comment:
I guess this should become a directory in the container rather than on the host? No need to mount it if it's temporary files (there will be more volumes in other places)